docker/dockerfiles/artifacts/artifact.Dockerfile

```diff
@@ -51,4 +51,10 @@ COPY --from=fe-builder /build/starrocks/output /release/fe_artifacts
 COPY --from=be-builder /build/starrocks/output /release/be_artifacts
 COPY --from=broker-builder /build/starrocks/fs_brokers/apache_hdfs_broker/output /release/broker_artifacts
 
+
+# Get ddprof
+RUN wget https://github.com/DataDog/ddprof/releases/download/v0.14.1/ddprof-0.14.1-amd64-linux.tar.xz -O ddprof-amd64-linux.tar.xz && \
```

Comment:
this won't work for arm platform. consider move to thirdparty so it will be available in non-container env?

Reply:
can you give me an example how to add it to thirdparty?
Thank you!